lib/chef/provider/systemd_unit.rb

```diff
@@ -227,10 +235,7 @@ def active?
       end
 
       def enabled?
-        # See https://github.com/systemd/systemd/blob/master/src/systemctl/systemctl-is-enabled.c
-        # Note: enabled-runtime is excluded because this is volatile, and the state of enabled-runtime
-        # specifically means that the service is not enabled
-        %w{enabled static generated alias indirect}.include?(systemd_unit_status["UnitFileState"])
+        systemd_unit_status["is-enabled"] == "enabled"
```

Comment:
there's some concerns around if omitting `static generated alias indirect` now creates a regression and we need to hear back from other customers if that would affect them.